gb-dbg/src/debugger/breakpoints.rs

```diff
@@ -95,8 +91,18 @@ impl BreakpointEditor {
         }
     }
 
-    fn is_valid_address(&self, address: &str) -> bool {
-        address.len() == 4 && u16::from_str_radix(address, 16).is_ok()
+    fn is_valid_address(address: &str) -> bool {
+        !address.is_empty() && u16::from_str_radix(address, 16).is_ok()
+    }
+
+    pub fn are_breakpoints_trigger(&mut self, pc: u16) -> bool {
```

Comment:
`is_breakpoint_triggered`

Reply:
There is more than one breakpoint that can be triggered